### ISSUE TITLE
Names mismatch

### ISSUE THREAD

**Issue by kokun-7** [Templarian/MaterialDesign#1442]
It appears that the names of the source google's icons are not the same in the MaterialDesign icons set.
This creates unnecessary confusion.  They really should be the same to make switching easier.

**Comment by CoDEmanX** [comment]
See https://github.com/Templarian/MaterialDesign/issues/1402#issuecomment-237312736 and https://github.com/Templarian/MaterialDesign/issues/678#issuecomment-139907567 why. 
Would be nice though if MDI could be used as a drop-in replacement. But it would mean to either provide yet another MDI flavor (regular + designer + compat), or to break backward compatibility and rename most icons (and potentially break it again in future when Google adds new, conflicting icons).

**Comment by JamesCoyle** [comment]
Can we have multiple names for just the Google icons? That way they can have mdi naming but be compatible. Would add a little bit to the CSS selectors of the web font but seems worth an extra 15-20 chars per icon if it improves compatibility.

**Comment by Templarian** [comment]
This project existed before the official set was released, so yes the naming they went with was not noun based (we followed Font Awesome's lead and used nouns and very similar naming for a lot of the icons).

Will probably not be adding any sort of compatibility mode as generally I won't be adding every icon (usually just adding them as they are requested 'missing') and the hope is people start with the community version and not the official set if they need the extra icons. This is also another maintenance feature that would have to be tracked.
